recipe/bld.bat

```diff
@@ -10,13 +20,13 @@ for %%d in (2 3 RZ) do (
         -DCMAKE_BUILD_TYPE=RelWithDebInfo     ^
         -DCMAKE_C_COMPILER=clang-cl           ^
         -DCMAKE_CXX_COMPILER=clang-cl         ^
+        -DCMAKE_AR=%BUILD_PREFIX%/bin/llvm-ar.exe   ^
+        -DCMAKE_RANLIB=%BUILD_PREFIX%/bin/llvm-ranlib.exe   ^
```

Comment:
Since `BUILD_PREFIX` also contains backwards slashes:
```suggestion
        -DCMAKE_AR=llvm-ar.exe   ^
        -DCMAKE_RANLIB=llvm-ranlib.exe   ^
```